### PR TITLE
Choose between item commands and item updates in Tasker plugin

### LINKED ISSUE #1620: Choose between item commands and item updates in Tasker plugin
See https://community.openhab.org/t/android-app-sends-alarm-time-as-long-not-datetime/83758/8

### PR DESCRIPTION
**Pull request by mueller-ma**
Fixes #1620

Signed-off-by: mueller-ma <mueller-ma@users.noreply.github.com>

### CODE REVIEW

File: mobile/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -333,7 +333,8 @@
     <string name="item_picker_custom">Custom</string>
     <string name="item_picker_dialog_title">Select state</string>
     <string name="item_picker">openHAB Items</string>
-    <string name="item_picker_blurb">Set \"%1$s\" (%2$s) to \"%3$s\"</string>
+    <string name="item_picker_blurb_command">Command \"%1$s\" (%2$s) to \"%3$s\"</string>
```
Comment (maniac103) [inline mobile/src/main/res/values/strings.xml]: `Send command \"%3$s\" to \"%1$s\" (%2$s)`?

File: mobile/src/main/java/org/openhab/habdroid/ui/TaskerItemPickerActivity.kt [hunk 2]
```diff
@@ -61,5 +66,6 @@ class TaskerItemPickerActivity(
         const val EXTRA_ITEM_LABEL = "itemLabel"
         const val EXTRA_ITEM_STATE = "itemState"
         const val EXTRA_ITEM_MAPPED_STATE = "itemMappedState"
+        const val EXTRA_ITEM_COMMAND = "command"
```
Comment (maniac103) [inline mobile/src/main/java/org/openhab/habdroid/ui/TaskerItemPickerActivity.kt]: `EXTRA_ITEM_SEND_COMMAND`? Otherwise one could think it's a string.

**Comment by maniac103** [comment]
> I'm not sure what's the best way for the user to change between item commands and item updates.

Maybe extend the state selection dialog to also include a radio button set which allows that selection?

**Comment by mueller-ma** [comment]
I also need to show the contact items again and add OPEN and CLOSED to the suggested commands.

**Comment by mueller-ma** [comment]
The command/update buttons aren't centered on some higher api level. I'll look into that.

**Pull request merged**